### PR TITLE
Feat: Add Majik Kids case study

### PR DESCRIPTION
**Pull request by adityaoberai**
## What does this PR do?

Adds Majik Kids case study

## Test Plan

- `pnpm i && pnpm run dev`
- Visit `/blog/post/case-study-majik-kids`

## Related PRs and Issues

No

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

File: src/routes/blog/post/case-study-majik-kids/+page.markdoc [hunk 1]
```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: "Majik Kids: Building an audio-based content platform for children"
+description: Learn how Appwrite Cloud simplified their development process and accelerated productivity for Majik Kids as they worked 100s of artists, including musicians, voice artists, illustrators, etc., to build an alternative content platform with a focus on audio storytelling, creating a healthier learning experience for children.
+date: 2024-01-19
+cover: /images/blog/majik-kids.png
+timeToRead: 5
```
Comment (gewenyu99) [inline src/routes/blog/post/case-study-majik-kids/+page.markdoc:7]: This is not a 5 minute read haha

File: src/routes/blog/post/case-study-majik-kids/+page.markdoc [hunk 2]
```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: "Majik Kids: Building an audio-based content platform for children"
+description: Learn how Appwrite Cloud simplified their development process and accelerated productivity for Majik Kids as they worked 100s of artists, including musicians, voice artists, illustrators, etc., to build an alternative content platform with a focus on audio storytelling, creating a healthier learning experience for children.
```
Comment (gewenyu99) [inline src/routes/blog/post/case-study-majik-kids/+page.markdoc:4]:
```suggestion
description: Learn how Appwrite Cloud simplified their development process and accelerated productivity for Majik Kids as they worked 100s of artists, including musicians, voice artists, and illustrators, to build an alternative content platform focusing on audio storytelling, creating a healthier learning experience for children.
```

Personal preference but I really dislike etc. in the middle of a sentence. It's also not necessary here to convey meaning

File: src/routes/blog/post/case-study-majik-kids/+page.markdoc [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+date: 2024-01-19
+cover: /images/blog/majik-kids.png
+timeToRead: 5
+author: aditya-oberai
+category: case-studies
+---
+
+# Ideating an alternative content platform for children
+
+Being an entrepreneur in the entertainment-based education space for over 15 years and a father of a child of age 7, Bradley T. Morris from Salt Spring Island, Canada, recognized the harmful effects that excessive screen time and screen addiction can have on the nervous system of his child. He and his wife had discovered already that engaging their son, Sauryn, with audio stories while allowing them to participate in activities such as drawing, painting, etc., did not cause similar withdrawal symptoms that screen addiction causes.
```
Comment (gewenyu99) [inline src/routes/blog/post/case-study-majik-kids/+page.markdoc:14]:
```suggestion
Being an entrepreneur in the entertainment-based education space for over 15 years and a father of a child of age 7, Bradley T. Morris from Salt Spring Island, Canada, recognized the harmful effects that excessive screen time and screen addiction can have on the nervous system of his child. He and his wife had discovered already that engaging their son, Sauryn, with audio stories while allowing them to participate in activities such as drawing or painting did not cause similar withdrawal symptoms that screen addiction causes. 
```

Same here. "Such as", "including", "as well as" are all phrases that mean "etc.", so we can skip it here.

File: src/routes/blog/post/case-study-majik-kids/+page.markdoc [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+
+## Leveraging different Appwrite products
+
+Keeping Appwrite as their backend, the engineering team started developing the Majik Kids app. Since the application primarily featured audio content in numerous manners, they were required to use a substantial number of binary files, such as MP3 files for the audio, image files for content covers, and PDF files for activity books. **Appwrite Storage** allowed them to quickly host all of these files, set permissions, and leverage the file URLs to consume the content easily.
+
+Phil also developed a sustainable workflow to quickly add any artist’s works (audio story, cover artwork, etc.) to the application, thanks to Appwrite. Since Appwrite offers **SDKs for both client and server-side languages**, including Python, Phil was able to develop a custom Python script that downloads the files from the artist’s shared Google Drive, verifies the integrity of the files, makes any necessary file type conversions, and uses the Appwrite Python SDK to upload all the files to Appwrite Storage.
```
Comment (gewenyu99) [inline src/routes/blog/post/case-study-majik-kids/+page.markdoc:55]: Not a review comment, but this is a correct use of etc. by comparison 😆

**Review by gewenyu99 — CHANGES_REQUESTED** [review]
Minor suggestions, rest looks fine <3

File: src/routes/blog/post/case-study-majik-kids/+page.markdoc (hunk 1 above)
Comment (adityaoberai) [reply src/routes/blog/post/case-study-majik-kids/+page.markdoc:7]: A fit it into an online time-to-read calculator and it gave <6 xD

Updating it to 7

File: src/routes/blog/post/case-study-majik-kids/+page.markdoc (hunk 3 above)
Comment (adityaoberai) [reply src/routes/blog/post/case-study-majik-kids/+page.markdoc:14]: Ah good point.

File: src/routes/blog/post/case-study-majik-kids/+page.markdoc (hunk 4 above)
Comment (adityaoberai) [reply src/routes/blog/post/case-study-majik-kids/+page.markdoc:55]: Haha thanks xD

**Review by gewenyu99 — APPROVED** [review]

**Pull request merged**